### PR TITLE
set up express

### PR DESCRIPTION
**Pull request by JunjiaWangUSF**
create dependency and give a instruction on how to set up backend with express

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,19 @@
...
+if there's a problem 
+ls ~/.bun/bin
+vim  ~/.bash_profile
+past following 
+export PATH="$HOME/.bun/bin:$PATH"
+source ~/.bashrc
+
+install node modules/express
+
+in root: cd backend && bun install
```
Comment (jcai28) [inline README.md:19]: Could we also mention how to start the backend in local

**Review by jcai28 — COMMENTED** [review]
Looks good to me, add more read me.

File: README.md (hunk 1 above)
Comment (JunjiaWangUSF) [reply README.md:19]: bun install will install all dependency, then we can use normal procedure like node app.js to start express.  I will add more clear instruction.

Comment (JunjiaWangUSF) [reply README.md]: pushed a clear instruction

**Review by jcai28 — APPROVED** [review]
looks good to me.

**Pull request merged**